frontend/packages/ux-editor/src/components/Elements/Elements.tsx

```diff
@@ -9,17 +9,27 @@ import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmen
 import classes from './Elements.module.css';
 
 import { useCustomReceiptLayoutSetName } from 'app-shared/hooks/useCustomReceiptLayoutSetName';
+import { useProcessTaskTypeQuery } from '../../hooks/queries/useProcessTaskTypeQuery';
 
 export const Elements = () => {
   const { org, app } = useStudioEnvironmentParams();
   const { selectedFormLayoutSetName, selectedFormLayoutName } = useAppContext();
+  const { data: processTaskType } = useProcessTaskTypeQuery(org, app, selectedFormLayoutSetName);
   const existingCustomReceiptName: string | undefined = useCustomReceiptLayoutSetName(org, app);
 
   const hideComponents =
     selectedFormLayoutName === 'default' || selectedFormLayoutName === undefined;
 
   const t = useText();
 
+  const shouldShowConfPageToolbar = () => {
+    return existingCustomReceiptName === selectedFormLayoutSetName || processTaskType === 'payment';
+  };
+
+  const confPageToolbarMode = () => {
+    return existingCustomReceiptName === selectedFormLayoutSetName ? 'receipt' : 'payment';
+  };
```

Comment:
It's not a big deal, but these methods can be converted to variables instead. This way, you won't need to invoke the method on line 43.

```suggestion
const shouldShowConfPageToolbar = existingCustomReceiptName === selectedFormLayoutSetName || processTaskType === 'payment';

const confPageToolbarMode = existingCustomReceiptName === selectedFormLayoutSetName ? 'receipt' : 'payment';

```